src/index.html

```diff
@@ -21,7 +22,308 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a class="header__logo-link" href="#">
+        <img class="header__logo-img"
+        src="./images/logo.png"
+        alt="logo" />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link nav__link--active is-active" href="#">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link nav__link--hover" href="#"
+            data-qa="nav-hover">
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="grid-container">
+
+      <div class="card"
+      data-qa="card">
```

Comment:
The `data-qa="card"` attribute should only be added to the first card. Currently, it is applied to all cards, which does not meet the task requirements. Please ensure this attribute is only present on the first card.